### DISCUSSION TITLE
Ability to filter out maps that don't give PP

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#22013] [Ideas] [answered]
It'd be nice if there was a filter so I can only see beatmaps that give Performance Points. So, being able to filter out beatmaps that don't have a leaderboard (in the downloaded maps).

**Comment by mk56-spn** [comment]
That's two different requests?? Loved maps have leaderboards and give no PP

**Comment by yamplum** [comment]
You can use the `status=r` (for "ranked") and `status=a` (for "approved") search filters. I don't think there's any way to combine them into one query, though.

See https://osu.ppy.sh/wiki/en/Client/Interface#Search

**Comment by ghost** [comment, marked as answer]
This is already possible with the Date Ranked filter.
